editor/editor_inspector.cpp

```diff
@@ -3097,14 +3108,8 @@ void EditorInspector::update_tree() {
 		}
 	}
 
-	// Only hide plugins if we are not editing any object.
 	// This should be handled outside of the update_tree call anyway (see EditorInspector::edit), but might as well keep it safe.
```

Comment:
Not sure what this comment refers to exactly, but it probably shouldn't be in 2 places.

Reply:
This is a comment on `_clear()`.

https://github.com/godotengine/godot/blob/c374ce211c00279ac4a69aaaa12b5ff12d7cf6da/editor/editor_inspector.cpp#L4219